### ISSUE TITLE
Handling declarations of dependencies

### ISSUE THREAD

**Issue by ConsoleTVs** [wessberg/rollup-plugin-ts#56]
I have 2 NPM packages. 1 contains the core stuff and it's buint using the tsc command, so the generated dist is full of .js and .d.ts files. The other project does use this lib and imports the core lib, extending some class.

The generated output does not bundle the .d.ts files of the core lib, making the declaration file useless. For instance, this is the generated output:

```ts
declare class Chart extends Chartisan {
    canvas: HTMLCanvasElement;
    constructor(options: ChartisanOptions);
    protected onUpdate(response: JSON): void;
}
export { Chart };
```
As noted, it have no clue what Chartisan  or ChartisanOptions is because it's not declared anywhere, How can I solve this?

**Comment by ConsoleTVs** [comment]
Please note, this is what is produced by running the tsc compiler:

```
import '@chartisan/chartisan/dist/style.css';
import { ChartisanOptions, isChartisan, Chartisan } from '@chartisan/chartisan';
export declare class Chart extends Chartisan {
    canvas: HTMLCanvasElement;
    constructor(options: ChartisanOptions);
    protected onUpdate(response: JSON): void;
}
declare global {
    interface Window {
        Chartisan: isChartisan;
    }
}
```

**Comment by wessberg** [comment]
Hey there. Thanks for reporting this. It sounds related to https://github.com/wessberg/rollup-plugin-ts/issues/44.

I'm working on a potential fix on a separate branch. but in order to verify this, I would very much like a repro from you, or at the very least a copy of your Rollup config do I can set up a test case for it.

**Comment by ConsoleTVs** [comment]
Hello @wessberg , Thanks for the quick response. I understand this is your project and free time and I would like to provide you with all the needed info to fix this since it's very important for me to make this work. Here's the repos: https://github.com/Chartisan

Chartisan repo is the core one, and chartjs the one using this lib. 

Thanks in advance, I'm kindly waiting for news on this!

**Comment by wessberg** [comment]
Thanks for the repo links! I will make sure to address it on the [experiment](https://github.com/wessberg/rollup-plugin-ts/tree/experiment?files=1) branch that I'm actively working on at the moment.

**Comment by wessberg** [comment]
This has been fixed as part of [v1.2.0](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.2.0). Happy New Year!
